### PR TITLE
Make the geojson with ruby since it handles yaml natively

### PR DESCRIPTION
**Pull request by crccheck**
and more importantly, it seems to retain the order of the hash items, hooray for idempotency

### CODE REVIEW

File: ripley.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+require 'json'
+require 'yaml'
+
+
+def format_geojson_from_docs(docs)
+    out = {
+        type: 'FeatureCollections',
+        features: [],
+    }
+    docs.each {|doc|
```
Comment (malev) [inline ripley.rb]: Use a `do` block here

File: ripley.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        coordinates = doc.delete('coordinates')
```
Comment (malev) [inline ripley.rb]: you don't need the ugly parenthesis

File: ripley.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        if coordinates
+            out[:features].push({
```
Comment (malev) [inline ripley.rb]: you don't need parenthesis or curly braces

Comment (crccheck) [inline ripley.rb]: you're going to have to show me this one. I can delete the curly braces, but when I delete the parens, I get a syntax error

**Pull request merged**